cdap-app-templates/cdap-etl/cdap-data-pipeline/src/test/java/co/cask/cdap/datapipeline/DataPipelineTest.java

```diff
@@ -327,7 +327,8 @@ public void testExternalSparkProgramPipelines() throws Exception {
         }
       }
     }
-    Assert.assertEquals(expectedLines, actualLines);
+    Assert.assertEquals(expectedLines.size(), actualLines.size());
```

Comment:
Thinking about this later don't think this will work in all case. 
It will fail in the case: 
expected: x, x, y
actual: x y y

Will fix it.

Reply:
should just change the lists to sets. All we really need to test is that the program runs and filters out some words.